ostree/ostree_transport.go

```diff
@@ -0,0 +1,171 @@
+package ostree
```

Comment:
(Due to the impact on signatures we generally want as close to 100% code coverage for the `ImageTransport` and `ImageReference` implementations as possible, but that will obviously wait until the formats and semantics are nailed down—and I can write that.)